model/adjuster/ip_tag.go

```diff
@@ -28,13 +28,22 @@ import (
 	"github.com/uber/jaeger/model"
 )
 
+var ipTagsToCorrect = map[string]bool{
```

Comment:
nit: I usually have my sets be map[string]struct{} because this takes up no extra memory compared to a bool.